content/posts/marimo.md

```diff
@@ -0,0 +1,271 @@
+
+```bash
+pip install marimo && marimo tutorial intro
+```
+
+at your command line.
+
+
+## Using Pyodide to make marimo accessible to everyone
+
+We decided to port marimo to Pyodide for three big reasons.
+
+- make scientific computing accessible to everyone by eliminating the need to
+  install and manage Python;
+- make it extremely easy to share executable notebooks by eliminating
+  the financial and technical burden of deploying backend infrastructure;
+- provide a delightfully snappy development experience by eliminating
+  network requests to a remote Python runner.
+
+Our motivation for combining marimo with Pyodide echoes the vision articulated
+by the developers of [Iodide](https://hacks.mozilla.org/2019/03/iodide-an-experimental-tool-for-scientific-communicatiodide-for-scientific-communication-exploration-on-the-web/), the experimental in-browser notebook for which
+Pyodide was originally created. While Iodide is no longer developed, we’re
+extremely grateful that it ushered Pyodide into existence.
+
+**The Pyodide-marimo playground.**
+We have developed a [Pyodide-powered marimo playground](https://marimo.new)
+that makes it extremely easy for anyone in the world to tinker with marimo
+notebooks. With just one click, playground notebooks can be shared with others,
+and conversely every local notebook can be shared via our playground.
+
+Here are some use cases that our Pyodide-powered notebooks enable:
+
+- Rapidly experiment with code and models.
+- Author blog posts, tutorials, and educational materials.
+- Embed interactive notebooks in library documentation or other static web pages.
+- Build and share tools like a data labeler, embedding visualizer, model
+  comparison tool, or anything else you can imagine.
+- Create and share bug reproductions.
+
+You can learn more about the playground, including how
+to embed marimo notebooks in static HTML pages, [at our
+docs](https://docs.marimo.io/guides/wasm.html).
+
+We’ve created a few example
+notebooks to jump-start your imagination. There’s one that [trains a tiny
+neural network](https://marimo.app/l/xpd4te), another that visualizes [Bayes’
+Theorem](https://marimo.app/l/odhwnq), and another that plots attractors of
+[dynamical systems](https://marimo.app/l/c7h6pz). One of our community members
+even created a [QR code
+generator](https://vxlabs.com/2024/03/02/contact-qrcode-generator-with-marimo-and-wasm/).
+
+When you take a moment to think about all this, it’s really quite *magical* ✨.
+Thanks to Pyodide, you can use any pure Python package, as well as a number of
+packages for scientific computing, including Pandas, NumPy, SciPy, matplotlib,
+and scikit-learn, entirely in the sandbox of your web browser --- _without
+installing Python on your machine and without paying for infrastructure_.
+
+## Implementation: Porting marimo to Pyodide
+
+Porting marimo to work with Pyodide was relatively straightforward, a testament to the
+latter's robustness and thorough documentation.
```

Comment:
Very kind! I _love_ to hear our documentation described as "thorough".

Reply:
It's true!